pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -382,6 +383,16 @@ func getDiskTargetsForMigration(dom cli.VirDomain, vmi *v1.VirtualMachineInstanc
 	return copyDisks
 }
 
+func loopbackAddress() string {
+	ipString := os.Getenv("KUBERNETES_SERVICE_HOST")
```

Comment:
I'm not sure we can use "KUBERNETES_SERVICE_HOST" for this purpose. Maybe we can add "MY_POD_IP" env to the virt-launcher. Please consult with @rmohr about it.

Reply:
i think that someone the env already have MY_POD_IP, 
i saw you are using it in launcher, just waiting because saw as well Roman asked some refactor there

Reply:
I'm using it in virt-handler, not launcher (although the code is in virt-launcher it is running in virt-handler. Confusing, I know).